packages/fxa-content-server/tests/functional/pages.js

```diff
@@ -7,35 +7,21 @@
 const { registerSuite } = intern.getInterface('object');
 var url = intern._config.fxaContentRoot;
 
-// Commented out pages here likely mean the React version is rolled out to 100%
-// and we don't want to check these for `#stage header` selectors.
```

Comment:
Might leave this comment as-is since we will be likely commenting out other tests here as we roll these out to 100% but prior to removing this code.